examples/plugins/job-pre-post/mail/setup.py

```diff
@@ -2,14 +2,19 @@
 
 name = 'avocado_job_mail'
 klass = 'Mail'
-entry_point = '%s = %s:%s' % (name, name, klass)
+entry_point = '%s = %s:%s' % ('job_mail', name, klass)
```

Comment:
I don't think this was intended for inclusion, was it? (just this line, the rest is valid)